fre/cmor/cmor_yamler.py

```diff
@@ -157,14 +162,27 @@ def cmor_yaml_subtool(yamlfile=None, exp_name=None, platform=None, target=None,
             fre_logger.info('indir = %s', indir)
 
             fre_logger.info('PROCESSING: ( %s, %s )', table_name, component)
-            cmor_run_subtool(
-                indir = indir ,
-                json_var_list = json_var_list ,
-                json_table_config = json_table_config ,
-                json_exp_config = json_exp_config ,
-                outdir = cmorized_outdir ,
-                run_one_mode = True ,
-                opt_var_name = None
-            )
+            if not dry_run_mode:
+                cmor_run_subtool(
+                    indir = indir ,
+                    json_var_list = json_var_list ,
+                    json_table_config = json_table_config ,
+                    json_exp_config = json_exp_config ,
+                    outdir = cmorized_outdir ,
+                    run_one_mode = True ,
+                    opt_var_name = None
+                )
```

Comment:
I think I see how you are working this. `grid_label` and `grid` are set in the cmor yaml, and `fre cmor run` replaces those two attributes in whatever experiment JSON is passed in.

Nice! I think this will work, but we need one more attribute; `nominal_resolution` that can be passed in just like `grid`. I don't think there will be any others.

I think I see how `fre cmor run` will accept the optional `grid` and `grid_label` options and use them instead,  and the `cmor_run_subtool` equivelant is called here. But I don't see where it's grabbing `grid` and `grid_label` from the yaml.